source/examples/rapids-sagemaker-higgs/notebook.ipynb

```diff
@@ -26,8 +26,10 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !docker tag {estimator_info['ecr_image']} {ECR_container_fullname}
</pre><p> </p><p>Re-tagging <code>estimator_info['rapids_container']</code> meant that this notebook was trying to run training in the <code>rapidsai/core</code> image, not the new image that we just built from it a few cells up.  </p><p> </p><p>This fixes that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/381/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>